strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileSystemProvider.java

```diff
@@ -290,7 +290,7 @@ protected void doDeletePath(RepositoryPath repositoryPath,
                    trashPath.getTarget(),
                    StandardCopyOption.REPLACE_EXISTING);
         
-        if (force && repository.allowsForceDeletion())
+        if (force && repository.isAllowsDelete())
```

Comment:
If I remember correctly it was exact **force  delete** feature, not just **delete**. 
@carlspring maybe you should note this.

Reply:
will check this out later

Reply:
Guys, I like the original name better. Gramatically, `isAllows` is not corect. Either `allowsDelete()`, or something else along these lines.

Reply:
corrected